### PR TITLE
Describe development process

### PR DESCRIPTION
**Pull request by shuheiktgw**
Describe and update development process

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,8 +35,18 @@ In addition, you may need to set the following environment variables in your env
...
+
+### Install nose-launchable locally 
+If you want to see whether your changes work as expected,  run the following command to install the plugin locally.
 
 ```bash
-$ python setup.py develop
+$ python setup.py install
 ``` 
+
+### Release nose-launchable
```
Comment (Konboi) [inline README.md:47]: We should run these commands at the main branch after merged a feature branch to the main branch, right?

Comment (shuheiktgw) [reply README.md:47]: Yes, that workflow was in my head when I wrote it down but you can actually include those changes into a feature branch if you want. It works in either way but `./scripts/bump_up.sh` creates a new branch so it does not work well with a feature branch...

Comment (Konboi) [reply README.md:47]: Thank you, I'll try it

**Review by Konboi — APPROVED** [review]

**Comment by shuheiktgw** [comment]
Thank you!

**Pull request merged**